terraform/lib/dependabot/terraform/file_parser.rb

```diff
@@ -168,7 +168,7 @@ def build_terragrunt_dependency(file, details)
       # Full docs at https://www.terraform.io/docs/modules/sources.html
       def source_from(details_hash)
         raw_source = details_hash.fetch("source")
-        bare_source = get_proxied_source(raw_source)
+        bare_source = Helpers::get_proxied_source(raw_source)
```

Comment:
```suggestion
        bare_source = Helpers.get_proxied_source(raw_source)
```

Ruby 🤷 (it's idiomatic to only use `::` for constant lookup and not methods), a linter might also complain about this, in which case apologies for the noise

Reply:
Fixed, it's been a while since I've done much ruby :)